library/std/src/io/error.rs

```diff
@@ -178,6 +178,12 @@ pub enum ErrorKind {
     /// read.
     #[stable(feature = "read_exact", since = "1.6.0")]
     UnexpectedEof,
+
+    /// This operation is unsupported on this platform.
+    ///
+    /// This means that the operation can never succeed.
+    #[stable(feature = "unsupported_error", since = "1.52.0")]
```

Comment:
Nightly is at 1.53.0 now:
```suggestion
    #[stable(feature = "unsupported_error", since = "1.53.0")]
```